engine/api/src/org/apache/cloudstack/engine/subsystem/api/storage/TemplateInfo.java

```diff
@@ -25,4 +25,6 @@
     String getUniqueName();
 
     String getInstallPath();
+
+    boolean isDirectDownload();
```

Comment:
Just for your info, be aware of not mixing `Boolean` and `boolean` types since the JVM won't check for those kind of mismatch as it does auto casting to the expected type. But you will end up with a NPE when getting a `Boolean` NULL on a method expecting a `boolean`.

Reply:
Thanks